src/hooks/useSearch.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+function useSearch() {
+  const getSearchResult = async (keyword, pageNum) => {
```

Comment:
```suggestion
  const getSearchResult = async (keyword, pageNum = 1) => {
```
**Recommend-Must:**: default 파라미터 적용할 수 있을 것 같습니다!

Reply:
좋은 제안 감사합니다 적용했어요!